com.unity.render-pipelines.core/Editor/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] // permit using internal interfaces with Moq
+[assembly: InternalsVisibleTo("Unity.RenderPipelines.Core.Editor.Tests")]
+[assembly: InternalsVisibleTo("Unity.RenderPipelines.Universal.Editor")] // remove as needed when API can be made public
+[assembly: InternalsVisibleTo("Unity.RenderPipelines.UniversalUpgradeProjectTests")]
```

Comment:
Imo we should avoid doing this. If AnimationClip API is not public and for now the scope is only to be used for URP, one possibility could be to make it internal  to `Universal.Editor` assembly and once by the time we look at it for HDRP we move to core and make API public.

Also, regarding the UniversalUpgradeProjectTest, how does that change based on the fact that we decided to integrate the animation clip upgrader with existing material upgrader?

Reply:
Changed in https://github.com/Unity-Technologies/Graphics/pull/4135/commits/5aa6524419a08dd468f9507f56f10ce9c2aea806

For now, I have given the test project visibility of internals for the `Universal.Editor` assembly. What is in there now is not a "true" end-to-end test since we have merged everything into the existing menu commands, but it is tests for the whole AnimationClip stage of the upgrade using actual assets, instead of mocks.